algosdk/future/transaction.py

```diff
@@ -489,6 +515,8 @@ def dictify(self):
             d["votelst"] = self.votelst
         if self.nonpart is not None:
             d["nonpart"] = self.nonpart
+        if self.state_proof_ID is not None:
+            d["sprfkey"] = self.state_proof_ID
```

Comment:
Can you treat this as a base64 encoded string, similar to `selkey` and `votepk`?